http/1.8/modules/ROOT/pages/http-request-ref.adoc

```diff
@@ -0,0 +1,696 @@
+			</tls:context>
+			<http:authentication >
+				<http:basic-authentication username="testuser" password="Mypassword" />
+			</http:authentication>
+		</http:request-connection>
+	</http:request-config>
+  <flow name="httplistenerresponseFlow1" >
+  <http:request method="GET" config-ref="HTTP_Request_configuration" path="/testpath"/>
+</flow>
+----
+
+== Configure HTTP Request Connectivity Testing
+
+The HTTP *Request* operation requires a global request configuration with a connection provider. Since HTTP Connector version 1.7.0, the connection provider supports connectivity testing.
+
+=== Configure Minimal Connectivity Testing
+
+To configure connectivity testing in the request global configuration without depending on the *Request* operation, add the `<http:connectivity-test>` and `requestPath` parameters in the `<http:request-connection>` tag as follows:
+
+[source,xml,linenums]
+----
+<http:request-config name="requestConfigWithOnlyPath">
+  <http:request-connection host="localhost" port="80" >
+    <http:connectivity-test requestPath="/thePath" />
+  </http:request-connection>
+</http:request-config>
+----
+
+After you configure the connectivity testing, you can test the connection in Studio:
+
+. Navigate to the *Global Elements* tab.
+. Select *HTTP Request configuration (Configuration)*.
+. Click *Edit* to open the global configuration window.
+. Click *Test connection...* to trigger a request to the URL `+http://localhost:80/thePath+`.
+
+If a connection is established, a request is sent, and a response is received. The test connection is successful if the response status code is in the range of 0-399. Otherwise, the test connection failed.
+
+
+=== Configure Full Connectivity Testing
+
+The connectivity testing `<http:connectivity-test>` requires only the `requestPath` parameter, though you can also configure additional parameters, as follows:
+
+[source,xml,linenums]
+----
+<http:connectivity-test requestPath="/{pathUriParam}" requestMethod="POST"
+                        responseTimeout="30" responseTimeoutUnit="SECONDS"
+                        followRedirects="true">
+  <http:request-body>
+    <![CDATA[The Body]]>
+  </http:request-body>
+  <http:request-headers>
+    <http:request-header key="Key" value="Value1" />
+    <http:request-header key="Key" value="Value2" />
+  </http:request-headers>
+  <http:request-query-params>
+    <http:query-param key="Key1" value="Value1" />
+    <http:query-param key="Key2" value="Value2" />
+  </http:request-query-params>
+  <http:request-uri-params>
+    <http:uri-param key="pathUriParam" value="testPath" />
+  </http:request-uri-params>
+  <http:response-validator>
+    <http:success-status-code-validator values="200" />
+  </http:response-validator>
+</http:connectivity-test>
+----
+
+In the previous example, if you click on the *Test Connection...* button in the global configuration window, HTTP Connector sends a POST request to `+http://localhost:80/testPath?Key1=Value1&Key2=Value2+`.
```

Comment:
I think it's OK to say "In this example" since the paragraph is right after the example.